### DISCUSSION TITLE
Using LiteStar with SuperTokens?

### DISCUSSION THREAD

**Discussion by Spectryx** [litestar-org/litestar#1490] [Ideas]
Hello!

Would it be possible to use LiteStar with SuperTokens Authentication? Or does it need it owns framework like it has for FastAPI/Flask?
https://supertokens.com/docs/emailpassword/pre-built-ui/setup/backend

**Comment by Goldziher** [comment]
No clue, i've never heard of this. 

But since its a paid product - we are not going to add support for this from within Litestar. They are more than welcome to add support for us of course- and it shouldnt be hard.

**Reply by Spectryx** [reply]
Ahh, ok! Ive used it with fastapi and its a pretty decent and simple authentication backend. Its Opensource, you can self host it for free:)

**Reply by Goldziher** [reply]
I see. Ok, we can gladly assist with adding support for us on their SDK. Maybe open an issue ok their repo and tag this discussion?

**Reply by Spectryx** [reply]
Will do!
